src/components/views/settings/tabs/room/RolesRoomSettingsTab.tsx

```diff
@@ -237,6 +238,7 @@ export default class RolesRoomSettingsTab extends React.Component<IProps> {
             [EventType.RoomEncryption]: _td("Enable room encryption"),
             [EventType.RoomServerAcl]: _td("Change server ACLs"),
             [EventType.Reaction]: _td("Send reactions"),
+            [EventType.RoomRedaction]: _td("Redact messages"),
```

Comment:
the string should be `Remove`

![image](https://user-images.githubusercontent.com/2403652/150777380-701490d0-e00d-46cb-a2e4-1aeb958b2a38.png)